requirements/requirements.txt

```diff
@@ -95,3 +95,5 @@ github3.py==1.0.0a4
 django-bulk-update==1.1.10
 csiphash==0.0.4
 json-delta==2.0
+email_validator==1.0.1
```

Comment:
cause it checks if the email is deliverable (important for hubspot): https://github.com/JoshData/python-email-validator#overview. it's easy to have a fake email that passes the regex but isn't an actual account